pkg/resource/v1/certconfig/desired_test.go

```diff
@@ -1 +1,96 @@
 package certconfig
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
+	"github.com/giantswarm/micrologger"
+	clientgofake "k8s.io/client-go/kubernetes/fake"
+
+	"github.com/giantswarm/cluster-operator/pkg/cluster"
+	"github.com/giantswarm/cluster-operator/pkg/resource/v1/certconfig/key"
+)
+
+func Test_GetDesiredState_Returns_CertConfig_For_All_Managed_Certs(t *testing.T) {
```

Comment:
This only checks that names match with `managedCertificates`. I'm not sure if there's point in validating deeper details as it would just replicate all the `new*` functions.